doc/api.yml

```diff
@@ -943,6 +953,17 @@ components:
         creation_datetime:
           type: string
           format: date-time
+    Promotion:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+          example: promotion21
+        promotion_range:
```

Comment:
No.

Use dates intervals instead of a field.

For example : 

```
promotion_begin : 
   type : string
   format: date

promotion_end : 
   type : string
   format: date
```